### DISCUSSION TITLE
Iframe: Verhindern beim Klick durch Link Webseite geöffnet wird

### DISCUSSION THREAD

**Discussion by Supermario-21** [Zefau/ioBroker.jarvis#1875] [General]
Hallo! Ich hab den Regenradar als Iframe eingefügt. Gibt es eine Möglichkeit zu verhindern, dass der User durch Klick auf diese Internetseite gelangt. Vielleicht über die Sandbox?

Danke im Voraus

**Comment by Zefau** [comment]
Müsste ich via Feature Request implementieren. Theoretisch könnte ich ein transparentes Objekt vor das iFrame setzen, damit dieses die Klicks abfängt und das iFrame diese nicht erhält.

**Comment by Zefau** [comment]
Ich muss mich korrigieren, mittels `'pointer-events': 'none'` geht es ganz easy. Kann ich kurzfristig implementieren.

**Reply by Zefau** [reply]
via https://github.com/Zefau/ioBroker.jarvis/issues/1880

**Comment by Supermario-21** [comment]
Super, Danke schön!
